docs/explanation/archive.rst

```diff
@@ -0,0 +1,348 @@
+proposed
+~~~~~~~~
+
+This *pocket* is a :term:`Staging Environment` the :term:`Ubuntu` community can
+opt in to verify the stability of any updates before they get deployed to a broader
+range of consumers.
+
+| After the initial release of an :term:`Ubuntu` :ref:`ArchiveSeries`, this
+  *pocket* contains non-security-related updates to :term:`Packages <Package>`
+  in the :ref:`ArchivePockets_Release` *pocket* before they get uploaded to the
+  :ref:`ArchivePockets_Updates` *pocket*.
+| Before the initial release of an :term:`Ubuntu` :ref:`ArchiveSeries`, this
+  *pocket* contains non-security-related updates to :term:`Packages <Package>`
+  in the :ref:`ArchivePockets_Release` *pocket* before they get uploaded to the
+  :ref:`ArchivePockets_Release` *pocket*.
+
+.. _ArchivePockets_Backports:
+
+backports
+~~~~~~~~~
+
+This *pocket* contains :term:`Packages <Package>` the :term:`Ubuntu` :ref:`ArchiveSeries`
+was initially **NOT** released with.
+
+The article :doc:`/explanation/backports` provides more information on backporting software.
+
+.. important::
+
+    The *backports pocket* does not come with any security support guarantee.
+    The :term:`Ubuntu` Security Team does not update :term:`Packages <Package>`
+    in the *backports pocket*. The :term:`Ubuntu` community is responsible for
+    maintaining packages in backports with later patches for bug fixes and
+    security updates.
+
+.. _ArchiveSuite:
+
+Suite
+-----
+
+A combination of a :ref:`ArchiveSeries` and a :ref:`ArchivePockets`. For example:
+
++---------------------+----------------------+---------------------------------+
+| *Suite*             | :ref:`ArchiveSeries` | :ref:`Pocket <ArchivePockets>`  |
++---------------------+----------------------+---------------------------------+
+| ``jammy``           | ``jammy``            | :ref:`ArchivePockets_Release`   |
++---------------------+----------------------+---------------------------------+
+| ``jammy-security``  | ``jammy``            | :ref:`ArchivePockets_Security`  |
++---------------------+----------------------+---------------------------------+
+| ``jammy-updates``   | ``jammy``            | :ref:`ArchivePockets_Updates`   |
++---------------------+----------------------+---------------------------------+
+| ``jammy-proposed``  | ``jammy``            | :ref:`ArchivePockets_Proposed`  |
++---------------------+----------------------+---------------------------------+
+| ``jammy-backports`` | ``jammy``            | :ref:`ArchivePockets_Backports` |
++---------------------+----------------------+---------------------------------+
+
+You can see all active suites here: http://archive.ubuntu.com/ubuntu/dists/
+
+.. note::
+
+    The ``devel`` :ref:`ArchiveSeries` allways mirrors the :ref:`ArchiveSeries` with
+    the :term:`Codename` of the :term:`Current Release in Development`.
+
+.. _ArchiveComponents:
+
+Components
+----------
+
+*Components* are logical subdivisions or namespaces of the :term:`Packages <Package>`
```

Comment:
namespaces is used here as an abstract concept rather than a concrete technical reference Many programmers know namespaces from their programming languages like C++/C# or Java/Python/JS/TS modules.

Do you think this should have a glossary entry?

Reply:
Hmmm...maybe? If it's a common term that anyone getting involved in packaging should already know, then possibly not. I'll leave it to your judgement.

Reply:
Resolved with e23ad4092120f6d5ff3f0c61b6769ca1dd845016, I added a term definition for namespaces